custom_components/intesisbox/intesisbox.py

```diff
@@ -78,6 +80,15 @@ async def keep_alive(self):
         else:
             _LOGGER.debug("Not connected, skipping keepalive")
 
+    async def poll_ambtemp(self):
+        """Retrieve Ambient Temperature to prevent integration timeouts."""
+        while self.is_connected:
+            _LOGGER.debug("Sending AMBTEMP")
+            self._write("GET,1:AMBTEMP")
+            await asyncio.sleep(10)
```

Comment:
This did you intend to use the new method you added?

```suggestion
            await self.delay(10)
```

Reply:
I see the confusion here . I added the method with the intention of initially using it but found that it wasn't required when using `_writeasync` with its `await asyncio.sleep(1)`